website/content/docs/platform/k8s/consul-to-raft.mdx

```diff
@@ -0,0 +1,361 @@
+        volumeMounts:
+          - name: storage-migration
+            mountPath: "/vault/storage-migration"
+          - name: data
+            mountPath: "/vault/data"
+    volumeMounts:
+      - name: storage-migration
+        mountPath: "/vault/storage-migration"
+    volumes:
+      - name: storage-migration
+        configMap:
+          name: storage-migration
+    dataStorage:
+      enabled: true
+      size: "1Gi"
+    ha:
+      enabled: true
+      replicas: 1
+      raft:
+        enabled: true
+        config: |
+          ui = true
+          service_registration "kubernetes" {}
+
+          listener "tcp" {
+            address = ":8200"
+            cluster_address = ":8201"
+            tls_disable = 1
+          }
+
+          storage "raft" {
+            path = "/vault/data"
+            retry_join {
+              auto_join_scheme = "http"
+              auto_join = "provider=k8s"
+            }
+          }
+  EOF
+  ```
+
+  **Configuration notes**
+    - `storage “raft”` configuration to specify the path for the Raft DB (`/vault/data` by default), and any `retry_join` parameters in your original configuration.
+      - This example uses `auto_join` to automatically find Raft peers via the Kubernetes API.  See the [`retry_join`](/vault/docs/configuration/storage/raft#retry_join-stanza) for more information.
+    - `dataStorage` configuration in the Helm override values, to specify the parameters of the PVCs the Vault `StatefulSet` will create.
+    - `extraInitContainers` will start an init container mounting the storage migration ConfigMap and `data` volume, which it will then use to execute the storage migration.  
+    - `replicas: 1`
+      - This setting is temporary for the purposes of the migration.  A new Vault `StatefulSet` with one replica to confirm the init container completed the migration and unseal Vault using the new storage backend.
+
+1. Apply this configuration.
+
+  ```shell-session
+  $ helm install vault hashicorp/vault -f vault-migration-values.yml
+  ```
+  
+1. Review migration logs and unseal Vault
+  
+  ```shell-session
+  $ kubectl logs vault-0 -c vault-server-migration
+  ```
+  
+  ```shell-session
+  $ kubectl exec -it vault-0 -- vault operator unseal
+  Key                     Value
+  ---                     -----
+  Seal Type               shamir
+  Initialized             true
+  Sealed                  false
+  Total Shares            1
+  Threshold               1
+  Version                 1.14.8+ent
+  Build Date              2023-12-05T01:49:39Z
+  Storage Type            raft
+  Cluster Name            vault-cluster-68870bf8
+  Cluster ID              cd18c692-f2e3-77a5-fba3-28f06f41f375
+  HA Enabled              true
+  HA Cluster              https://vault-0.vault-internal:8201
+  HA Mode                 active
+  Active Since            2024-04-10T04:20:23.707098402Z
+  Raft Committed Index    157
+  Raft Applied Index      157
+  Last WAL                55
+  ```
```

Comment:
I would number the steps separately.

````suggestion
1. Review the migration logs.
  
  ```shell-session
  $ kubectl logs vault-0 -c vault-server-migration
  ```
  
1. Unseal Vault.

  ```shell-session
  $ kubectl exec -it vault-0 -- vault operator unseal
  Key                     Value
  ---                     -----
  Seal Type               shamir
  Initialized             true
  Sealed                  false
  Total Shares            1
  Threshold               1
  Version                 1.14.8+ent
  Build Date              2023-12-05T01:49:39Z
  Storage Type            raft
  Cluster Name            vault-cluster-68870bf8
  Cluster ID              cd18c692-f2e3-77a5-fba3-28f06f41f375
  HA Enabled              true
  HA Cluster              https://vault-0.vault-internal:8201
  HA Mode                 active
  Active Since            2024-04-10T04:20:23.707098402Z
  Raft Committed Index    157
  Raft Applied Index      157
  Last WAL                55
  ```
````